### PR TITLE
Changelog: 9.4.1

### PR DESCRIPTION
**Pull request by jeherve**
#### Changes proposed in this Pull Request:

* Add changelog for 9.4.1

#### Jetpack product discussion

* N/A

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

* Check for typos.

#### Proposed changelog entry for your changes:

* This is it.

### CODE REVIEW

File: projects/plugins/jetpack/readme.txt [hunk 1]
```diff
@@ -1,7 +1,7 @@
...
 Tags: WP, backup, social, AMP, WooCommerce, malware, scan, spam, CDN, social
-Stable tag: 9.3.1
```
Comment (sergeymitr) [inline projects/plugins/jetpack/readme.txt:4]: I'm not quite sure about the purpose of this tag, but according to the log it always seems to be one version behind.
Should this point to the previous version, or `9.4.1` is the correct value?

File: projects/plugins/jetpack/readme.txt [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
+Stable tag: 9.4.1
```
Comment (sergeymitr) [inline projects/plugins/jetpack/readme.txt:4]: I'm not quite sure about the purpose of this tag, but according to the log it always seems to be one version behind.
Should this point to the previous version, or `9.4.1` is the correct value?

Comment (jeherve) [reply projects/plugins/jetpack/readme.txt:4]: The tag isn't very useful here in our development environment. It is, however, useful in the stable version of Jetpack, delivered by WordPress.org. There, it always matches the current stable version. Hence the change to 9.4.1 now.

File: projects/plugins/jetpack/readme.txt [hunk 3]
```diff
@@ -241,6 +241,15 @@ Our Cookie and Consent Banner can help you comply with GDPR. The European Union
 
 = 9.4 =
```
[the remark this answers is not in the document]
Comment (jeherve) [reply projects/plugins/jetpack/readme.txt:242]: Yes indeed!

**Review by brbrr — APPROVED** [review]

**Comment by jeherve** [comment]
Cherry-picked to `jetpack/branch-9.4` in 3a43cc2ca74255c3510f44884560d3a6a98c38f9

**Pull request merged**